### PR TITLE
Fix ticker counter for low amounts

### PR DESCRIPTION
**Pull request by tomrf1**
## Why are you doing this?
The ticker gets stuck if its initial value is below 100 (it's initialised by `totalContributed * 0.8`).
This change makes it jump straight to `totalContributed` if that's the case.

## Screenshots
Screenshot 2019-05-24 at 11 10 36

### CODE REVIEW

File: support-frontend/assets/components/ticker/contributionTicker.jsx [hunk 1]
```diff
@@ -128,7 +128,8 @@ export default class ContributionTicker extends Component<PropTypes, StateTypes>
   increaseTextCounter = () => {
     const nextCount = this.state.count + Math.floor(this.state.count / 100);
 
-    if (this.dataFromServer.totalContributed && nextCount >= this.dataFromServer.totalContributed) {
+    if (this.dataFromServer.totalContributed &&
```
Comment (joelochlann) [inline support-frontend/assets/components/ticker/contributionTicker.jsx]: it's hard to tell just from the code what that `nextCount <= this.state.count` is actually doing - is there any way that variable names/comments could make it clear?

Comment (tomrf1) [reply support-frontend/assets/components/ticker/contributionTicker.jsx]: I've just pushed a commit that attempts to make it clearer

Comment (joelochlann) [reply support-frontend/assets/components/ticker/contributionTicker.jsx]: thanks!

File: support-frontend/assets/components/ticker/contributionTicker.jsx [hunk 2]
```diff
@@ -128,13 +128,17 @@ export default class ContributionTicker extends Component<PropTypes, StateTypes>
   increaseTextCounter = () => {
     const nextCount = this.state.count + Math.floor(this.state.count / 100);
 
-    if (this.dataFromServer.totalContributed && nextCount >= this.dataFromServer.totalContributed) {
+    const finishedCounting = this.dataFromServer.totalContributed &&
+      (nextCount <= this.state.count || //count isn't going up because total is too small
```
Comment (jlieb10) [inline support-frontend/assets/components/ticker/contributionTicker.jsx]: Nice!

**Comment by jlieb10** [comment]
Really great work figuring out this bug and with the naming and inline documentation @tomrf1 👍

**Pull request merged**